WeakAuras/Modernize.lua

```diff
@@ -1633,7 +1633,7 @@ function Private.Modernize(data)
 
   if data.internalVersion < 62 then
     if data.regionType == "dynamicgroup" then
-      if data.sort == "CUSTOM" and type(data.sortOn) ~= "string" then
+      if data.sort == "custom" and type(data.sortOn) ~= "string" then
```

Comment:
do you need another modernize to fix this up?

Reply:
Yeah you need to do that in a new modernize version, so that the code that we wrongly didn't convert gets converted.